.github/workflows/move-test-compiler-v2.yaml

```diff
@@ -0,0 +1,36 @@
+name: "Aptos Move Test for Compiler V2"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'aptos-move/e2e-move-tests/**'
```

Comment:
You want to add these yaml files themselves when they get changed as well that way it will trigger on this PR

Reply:
done